src/main/resources/default/templates/biz/jobs/job.html.pasta

```diff
@@ -124,6 +124,12 @@
                     } else {
                         _elem.setAttribute("hidden", true);
                     }
+                    if (param.clear) {
+                        _elem.dispatchEvent(new CustomEvent("clear"));
+                    }
+                    if (typeof param.updatedValue !== 'undefined') {
+                        _elem.dispatchEvent(new CustomEvent("updated-value", {detail: param.updatedValue}));
+                    }
```

Comment:
There is now a sirius method that allows to trigger custom events while supporting all browsers. it is called `sirius.dispatchEvent('event')'`. Maybe we could expand it to optionally also take a details object.